packages/stryker/README.md

```diff
@@ -151,12 +150,11 @@ In addition to requiring your test runner to be able to report the code coverage
  Currently, `stryker-mocha-runner` as well as `stryker-karma-runner` support this. However, `stryker-karma-runner` support is limited to using it with `Jasmine` as the test framework 
  (`Mocha` is not yet supported).
 
-#### Mutator 
-**Command line:** `--mutator es5`  
-**Config file:** `mutator: { name: 'es5', excludedMutations: ['BooleanSubstitution', 'StringLiteral'] }`  
-**Default value:** `es5`  
-**Mandatory**: no  
-**Description:**  
+### `mutator` [`object`]
+Default: `es5`  
+Command line: `--mutator es5`  
+Config file:  `mutator: { name: 'es5', excludedMutations: ['BooleanSubstitution', 'StringLiteral'] }`  
```

Comment:
In most scenario's the `mutator` property in a config file will be a string. [Example can be found here](https://github.com/stryker-mutator/stryker-handbook/blob/master/stryker/guides/angular.md#configuration).

How could we show that the mutator could both be an `object` and a `string`?

Reply:
Updated to use a '|' in the docs, what do you think?